From 53a8436893f4553cf4dc2af02b7d187f1b9a3b8b Mon Sep 17 00:00:00 2001
From: Didier Raboud <odyx@debian.org>
Date: Fri, 12 Dec 2014 12:59:43 +0100
Subject: [PATCH] Change ppd-updaters trigger to use 'interest-noawait' instead
 of 'interest' to avoid trigger cycles

Thanks-To: Guillem Jover <guillem@debian.org>
Closes: #772871
Closes: #771765
---
 debian/cups.triggers | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/cups.triggers b/debian/cups.triggers
index 7c73fe1..e741a7b 100644
--- a/debian/cups.triggers
+++ b/debian/cups.triggers
@@ -1,4 +1,4 @@
 # Trigger is activated when a driver touches a file named after the binary
 # package name containing proper DRIVER_REGEXP and GENNICKNAME_REGEXP shell
 # variables in the /usr/share/cups/ppd-updaters/ directory
-interest /usr/share/cups/ppd-updaters
+interest-noawait /usr/share/cups/ppd-updaters
-- 
2.1.3

